src/codeFlowDecorations.ts

```diff
@@ -346,38 +348,38 @@ export class CodeFlowDecorations implements Disposable {
 
     private static LocationDecorationType = window.createTextEditorDecorationType({
         dark: {
-            backgroundColor: "rgba(50,50,200,.5)",
+            backgroundColor: `rgba(50,50,200,.5)`,
         },
         light: {
-            backgroundColor: "rgba(50,50,200,.3)",
+            backgroundColor: `rgba(50,50,200,.3)`,
         },
-        overviewRulerColor: "blue",
+        overviewRulerColor: `blue`,
         overviewRulerLane: OverviewRulerLane.Left,
         rangeBehavior: DecorationRangeBehavior.ClosedClosed,
     });
 
     private static SelectionDecorationType = window.createTextEditorDecorationType({
-        borderStyle: "solid",
-        borderWidth: "1px",
+        borderStyle: `solid`,
+        borderWidth: `1px`,
         dark: {
-            borderColor: "white",
+            borderColor: `white`,
         },
         light: {
-            borderColor: "black",
+            borderColor: `black`,
         },
-        overviewRulerColor: "red",
+        overviewRulerColor: `red`,
         overviewRulerLane: OverviewRulerLane.Left,
         rangeBehavior: DecorationRangeBehavior.ClosedClosed,
     });
 
     private static UnimportantLocationDecorationType = window.createTextEditorDecorationType({
         dark: {
-            backgroundColor: "rgba(150,150,150,.4)",
+            backgroundColor: `rgba(150,150,150,.4)`,
```

Comment:
Single quotes?

Reply:
Yes. Fixed, everywhere

---
In reply to: [407679828](https://github.com/microsoft/sarif-vscode-extension/pull/235#discussion_r407679828) [](ancestors = 407679828)